# Disable separable compilation otherwise GCC gets passed cu.o files which it
# doesn't know how to handle.
diff --git a/CMakeLists.txt b/CMakeLists.txt
index ba53137..845132a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -132,7 +132,7 @@ if (MAGMA_ENABLE_CUDA)
     set( NV_SM    "" )
     set( NV_COMP  "" )

-    set(CUDA_SEPARABLE_COMPILATION ON)
+    set(CUDA_SEPARABLE_COMPILATION OFF)

     # nvcc >= 6.5 supports -std=c++11, so propagate CXXFLAGS to NVCCFLAGS.
     # Older nvcc didn't support -std=c++11, so previously we disabled propagation.
